src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./styles.css";
+import PokeCard from "./components/PokeCard.js";
+import { mockPokemonData } from "./mock/pokeData.js";
+
+export default function App() {
+  return (
+    <div className="App">
+      <h1>Welcome to your second mini challenge!</h1>
+
+      <PokeCard {...mockPokemonData} />
```

Comment:
This is fine and working but it is preferable to use one property at a time so that it can be more readable to others.
Also, you could pass extra fields not required in the component. Even if the component ignores them.